zipkin-cassandra/src/main/scala/com/twitter/cassie/connection/ClusterClientProvider.scala

```diff
@@ -141,7 +141,11 @@ private[cassie] class ClusterClientProvider(
         // call login
         val loginMap: java.util.Map[String, String] = HashMap("username" -> username, "password" -> password)
         val authRequest = new AuthenticationRequest(loginMap)
-        client.login(authRequest)
+        val future = client.login(authRequest)
+        future onFailure { exception =>
+          throw exception
+        }
```

Comment:
what is the thing you want to test? remember, your types will protect you from a lot of things that would normally need to be unit tested in ruby.